modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -542,8 +542,17 @@ This command also runs the suite through MUnit for the operation defined in the
 [INFO] ------------------------------------------------------------------------
 ➜  hello-smart-connector
 ----
-
-== Consuming Mule Plugin from an XML SDK Module
++
+(3) When you want to add this connector to your project from your local Maven repository for testing, you need only add the following dependency to use this connector from your Studio palette.
```

Comment:
"a project" to avoid distracting repetition? And delete "you need only" for concision without sacrificing friendliness? Also better to put the last clause as a separate sentence (I found it a bit hard to read all in one sentence)?